lib/tessel/deploy.js

```diff
@@ -436,7 +436,10 @@ actions.tarBundle = function(opts) {
         })
         .bundle(function(error, results) {
           if (error) {
-            return reject(error);
+            return reject({
```

Comment:
Because those are all taking the easy path that automatically logs with `logs.warn`, not `logs.err`. I explicitly wanted `logs.err` and this was the only way to force it to happen, because: https://github.com/tessel/t2-cli/blob/e7299f0b85862c01a10158ef92ec000ede42d07f/bin/tessel-2.js#L290-L307